tensorflow/lite/delegates/nnapi/acceleration_test_list.cc

```diff
@@ -331,6 +331,8 @@ ResizeNearestNeighborOpTest/ResizeNearestNeighborOpTest.+AlignCorners.*/0,30
 ResizeNearestNeighborOpTest/ResizeNearestNeighborOpTest.+HalfPixelCenters.*/0,30
 // Only models with constant size tensor are accelerated
 ResizeNearestNeighborOpTest/ResizeNearestNeighborOpTest/.+/0,29
+// 16-bit tests are not supported
+-ResizeNearestNeighborOpTest/.*Int16.*
```

Comment:
This should be `.+Int16` instead of `.*Int16.*`

Reply:
Thank you for the review, I made the change.

Reply:
Actually, we need to put the deny-listed line above the allow-listed rule and it is a parameterized test so please make it look like this:

```
// 16-bit tests are not supported
-ResizeNearestNeighborOpTest.+Int16/.+
// Only models with constant size tensor are accelerated
ResizeNearestNeighborOpTest/ResizeNearestNeighborOpTest/.+/0,29
```

Reply:
sorry for the troubles

Reply:
No problem, I made the change. Thanks!

Reply:
Hi,

You didn't make the full change. In my line above, the denylist line needs to be:

```
-ResizeNearestNeighborOpTest.+Int16/.+
```

whereas in your latest commit you still have:

```
-ResizeNearestNeighborOpTest.+Int16
```

The characters at the end are necessary due to the parameterized test.

Reply:
Oh sorry, I should have read your message more carefully. I made the change, thanks!